models/migrations/v136.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+	pull_service "code.gitea.io/gitea/services/pull"
+
+	"xorm.io/xorm"
+)
+
+func addCommitDivergenceToPulls(x *xorm.Engine) error {
+
+	if err := x.Sync2(new(models.PullRequest)); err != nil {
+		return fmt.Errorf("Sync2: %v", err)
+	}
+
+	var last int
+	batchSize := setting.Database.IterateBufferSize
+	sess := x.NewSession()
+	defer sess.Close()
+	for {
+		if err := sess.Begin(); err != nil {
+			return err
+		}
+		var results = make([]*models.PullRequest, 0, batchSize)
+		err := sess.Where("has_merged = ?", false).OrderBy("id").Limit(batchSize, last).Find(&results)
+		if err != nil {
+			return err
+		}
+		if len(results) == 0 {
+			break
+		}
+		last += len(results)
```

Comment:
A small performance optimiziation is to use `results.LoadIssues()` but not load issue in the loop.

Reply:
LoadIssues() for a PullList does not exist :(